src/Components/CreateImageWizard/formComponents/MountPoint.js

```diff
@@ -36,7 +36,7 @@ const MountPoint = ({ ...props }) => {
         return;
       }
     }
-  }, []);
+  }, [props.mountpoint]);
```

Comment:
Use this pattern instead:
https://react.dev/learn/you-might-not-need-an-effect#adjusting-some-state-when-a-prop-changes

Reply:
as we discussed in the group channel, we agreed to add those dependencies
and in the future maybe we will refector the code and remove the useEffect hook were ever we can